src/compile/RuleCompiler.java

```diff
@@ -1231,7 +1231,7 @@ private static void liftupActiveAtoms(Membrane mem)
 	 */
 	private void simplify() throws CompileException
 	{
-		staticUnify(rs.leftMem);
+	        staticUnify(rs.leftMem);
```

Comment:
```suggestion
		staticUnify(rs.leftMem);
```
タブ関連は元に戻しておいた方がいいでしょう。